### ISSUE TITLE
ONNX operators used in image classification models

### ISSUE THREAD

**Issue by kuke** [PaddlePaddle/Paddle2ONNX#50]
- recognize_digits: `Conv`, `Add`, `Relu`, `MaxPool`, `BatchNormalization`, `Reshape`, `MatMul`, `Softmax`, `Tanh`

- ResNet: `Conv`, `Add`, `Relu`, `AveragePool`, `Reshape`, `MatMul`, `Softmax`

- VGG: `Conv`, `Add`, `Relu`, `Dropout`, `Constant`, `Mul`, `MaxPool`, `Reshape`, `MatMul`, `BatchNormalization`, `Softmax`

- MobileNet: `Conv`, `BatchNormalization`, `Relu`, `GlobalAveragePool`, `Reshape`, `MatMul`, `Add`, `Softmax`

- SE_ResNeXt: `Conv`, `BatchNormalization`, `Relu`, `MaxPool`, `GlobalAveragePool`, `Reshape`, `MatMul`, `Add`, `Sigmoid`, `Mul`, `Dropout`, `Constant`, `Softmax`

- Inception V4: `Conv`, `BatchNormalization`, `MaxPool`, `Concat`, `AveragePool`, `Dropout`, `Constant`, `Mul`, `Reshape`, `MatMul`, `Add`, `Softmax`

ONNX == `1.0.1`, Data layout == `NCHW`
